validation/astra-sim-backends/analytical-vs-ns3-backend-validation.md

```diff
@@ -0,0 +1,21 @@
+
+# Analytical Backend vs NS-3 Backend Validation
+<!-- 
```

Comment:
Guess these comments are to be populated in next versions? Or we can just remove them.